### DISCUSSION TITLE
Question about cloud backups

### DISCUSSION THREAD

**Discussion by DrownedStarfish** [standardnotes/forum#2716] [General] [answered]
Hi!

I am using CloudLink to make automatic daily backups in one of my cloud accounts.  How do I set it do store the backups encrypted, or the opposite, to store them in plaintext?

Thanks.
D.

**Comment by JaspalSuri** [comment]
Hi @DrownedStarfish,

Currently, for ~security and privacy~ technical reasons, there isn't an option to export backups (via CloudLink) in plaintext. Users would have to manually backup decrypted backups of their notes via the desktop, web, or mobile apps through the respective app's settings page (which is accessed by the `Account` button on the web & desktop apps and the gear icon embedded within the left sidebar of the mobile app). 

Please let us know if you have any other questions. 🙂

**Comment by atmoio** [comment]
As a technical note, our servers are the ones that make the daily email backups, and our servers only have access to encrypted data. So plaintext backups is not technically feasible overall.

**Comment by DrownedStarfish** [comment, marked as answer]
Thanks, everyone.  I think that I understand the situation now.

Just to summarize for anyone else who might be interested:
If you want unencrypted backups (on Windows), you can get them by setting the Account|Data Backups control on your desktop UI of Standard Notes to Decrypted.  If you want a remote unencrypted backup, just do that and then make a remote backup of the local one by whatever method you normally use for remote backups.

Thanks again.
DS

**Comment by JaspalSuri** [comment]
You can use the [decrypt script from this repo](https://github.com/standardnotes/decrypt) to decrypt your encrypted notes. Feel free to store a copy of it somewhere adjacent to where your notes are being stored.
